packages/wallet-sdk/package.json

```diff
@@ -36,7 +36,6 @@
   "peerDependencies": {
     "@react-native-async-storage/async-storage": "*",
     "expo": "*",
-    "expo-linking": "*",
```

Comment:
Removing expo-linking -- We don't need to depend on this. Users might get their deeplinks from other places like Branch